docker/validator/setup.sh

```diff
@@ -10,7 +10,7 @@ go build \
     -X github.com/cosmos/cosmos-sdk/version.AppName=nymd \
     -X github.com/CosmWasm/wasmd/app.NodeDir=.nymd \
     -X github.com/cosmos/cosmos-sdk/version.Version=${WASMD_VERSION} \
-    -X github.com/cosmos/cosmos-sdk/version.Commit=1920f80d181adbeaedac1eeea1c1c6e1704d3e49 \
+    -X github.com/cosmos/cosmos-sdk/version.Commit=1d436638af7cacb5aeeb7248b57b085c64f3ae35 \
```

Comment:
Perhaps this hash should be defined alongside `WASMD_VERSION` since it really depends on it?

Reply:
Done :+1: